app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -4,7 +4,7 @@ class Api::V2::ActivationKeysController < Api::V2::ApiController
 
     before_action :verify_presence_of_organization_or_environment, :only => [:index]
     before_action :find_environment, :only => [:index, :create, :update]
-    before_action :find_optional_organization, :only => [:index, :create, :show]
+    before_action :find_optional_nested_object, :only => [:index, :create, :show]
```

Comment:
What is a nested object? I only ask cause this goes from obvious what it is doing to largely unreadable. I realize this is trying to re-use resources from Foreman core but I'd ask that we keep the obvious naming and wrap this call to provide a better interface.